### ISSUE TITLE
Recently introduced panic due to interface conversion

### ISSUE THREAD

**Issue by mgalgs** [achannarasappa/ticker#112]
**Describe the bug**
Panic due to interface conversion (see stack trace below)

**To Reproduce**
1. Check out `v3.1.2` (currently the tip of master) and run with `go run main.go -w TSLA`

**Expected behavior**
Normal execution of the program.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Linux
 - Terminal: Alacritty
 - Terminal Version: `alacritty 0.7.1 (a852eb85)`
 - ticker Version: `v3.1.2`

**Additional context**

```
$ go run main.go -w TSLA
Caught panic:

interface conversion: interface {} is nil, not map[string]position.Position

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack(0x73, 0x0, 0x0)
        /usr/lib/go/src/runtime/debug/stack.go:24 +0x9f
runtime/debug.PrintStack()
        /usr/lib/go/src/runtime/debug/stack.go:16 +0x25
github.com/charmbracelet/bubbletea.(*Program).Start.func1(0xc000620960)
        /home/mgalgs/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.12.2/tea.go:119 +0xd3
panic(0x91cde0, 0xc000378510)
        /usr/lib/go/src/runtime/panic.go:969 +0x1b9
github.com/achannarasappa/ticker/internal/position.GetPositions.func1(0xc0005da000, 0x1, 0x1, 0x20300000000000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/mgalgs/go/ticker/internal/position/position.go:147 +0x4eb
github.com/achannarasappa/ticker/internal/ui.Model.Update(0x5, 0xc00007dc40, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/mgalgs/go/ticker/internal/ui/ui.go:118 +0x4fa
github.com/charmbracelet/bubbletea.(*Program).Start(0xc000620960, 0x0, 0x0)
        /home/mgalgs/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.12.2/tea.go:214 +0x61c
github.com/achannarasappa/ticker/internal/ui.Start.func1(0x0, 0x0)
        /home/mgalgs/go/ticker/internal/ui/start.go:14 +0x1b6
github.com/achannarasappa/ticker/internal/cli.Run.func1(0xcfdd80, 0xc00000ea00, 0x0, 0x2)
        /home/mgalgs/go/ticker/internal/cli/cli.go:34 +0x2a
github.com/spf13/cobra.(*Command).execute(0xcfdd80, 0xc000020190, 0x2, 0x2, 0xcfdd80, 0xc000020190)
        /home/mgalgs/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xcfdd80, 0x44730a, 0xcb53c0, 0xc000000180)
        /home/mgalgs/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/mgalgs/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/achannarasappa/ticker/cmd.Execute()
        /home/mgalgs/go/ticker/cmd/root.go:31 +0x31
main.main()
        /home/mgalgs/go/ticker/main.go:6 +0x25
```

(I had to comment out the [`p.ExitAltScreen()`](https://github.com/achannarasappa/ticker/blob/027aaf39db4c2d231911bead65887c9fda4c5bfe/internal/ui/start.go#L15) to avoid having the stack trace disappear)

It was working fine on `v3.0.1` so I bisected (`git bisect start master v3.0.10`) and landed on:

```
de5a8460e703f7f76e9d7cdd30ac67ab14371ed5 is the first bad commit
commit de5a8460e703f7f76e9d7cdd30ac67ab14371ed5
Author: Ani Channarasappa <git@ani.dev>
Date:   Sun Feb 14 17:14:23 2021 -0500

    refactor: combined position and position summary calculations

 internal/currency/currency.go      |  11 ++-
 internal/currency/currency_test.go |  41 +++++++++-
 internal/position/position.go      |  82 +++++++++++--------
 internal/position/position_test.go | 161 ++++++++++++-------------------------
 internal/quote/quote.go            |   2 +-
 internal/ui/ui.go                  |   6 +-
 6 files changed, 150 insertions(+), 153 deletions(-)
```

**Comment by achannarasappa** [comment]
Thanks for reporting this - I believe I've resolved this in the latest release v3.1.4

**Comment by viperperidot** [comment]
Not sure if this is the same issue but I just had this happen running 3.1.3:

```
Caught panic:

interface conversion: interface {} is nil, not map[string]position.Position

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack(0x73, 0x0, 0x0)
	/opt/hostedtoolcache/go/1.15.8/x64/src/runtime/debug/stack.go:24 +0x9f
runtime/debug.PrintStack()
	/opt/hostedtoolcache/go/1.15.8/x64/src/runtime/debug/stack.go:16 +0x25
github.com/charmbracelet/bubbletea.(*Program).Start.func1(0xc0001d0bd0)
	/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.12.2/tea.go:119 +0xd3
panic(0x9179a0, 0xc000099020)
	/opt/hostedtoolcache/go/1.15.8/x64/src/runtime/panic.go:969 +0x1b9
github.com/achannarasappa/ticker/internal/position.GetPositions.func1(0xd32c68, 0x0, 0x0, 0x41582f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/runner/work/ticker/ticker/internal/position/position.go:158 +0x565

```

**Comment by mgalgs** [comment]
Yes!

Fixed-in: 8a8fd37796b3aae262e69348b80226a7594f647a

Thank you :pray:

**Comment by cwarden** [comment]
@achannarasappa do you want a separate issue for this one? https://github.com/achannarasappa/ticker/commit/2f674085#r47140876
